examples/uikit-app-shell/src/pages/Home/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from "./Home";
```

Comment:
Doesn't App Shell worth with `autoViewsAndRoutes` and `src/pages/*.tsx`? 🤔

Reply:
I wasn't able to make it work with `src/pages/*.tsx` but I can try again in case I did something wrong the first time around and ask the App Shell team about this

Reply:
I tried again but it still not working. I asked the App Shell about this, we'll see!